python/torch_mlir/dialects/torch/importer/jit_ir/build_tools/shape_lib_gen.py

```diff
@@ -979,6 +979,25 @@ def aten〇embedding_bag〇padding_idx(weight: List[int], indices: List[int], of
 
     return output_bag_shape, offset2bag_shape, bag_size_shape, max_indices_shape
 
+def aten〇_embedding_bag(weight: List[int], indices: List[int], offsets: List[int], scale_grad_by_freq: bool = False, mode: int = 0, sparse: bool = False, per_sample_weights: Optional[List[int]] = None, include_last_offset: bool = False, padding_idx: int = -1) -> Tuple[List[int], List[int], List[int], List[int]]:
```

Comment:
Is there a reason this shape function is simpler than the `embedding_bag_padding_idx` one?

Reply:
yes since the mode is always = 0 for this and other some other parameters are constants..

Reply:
Isn't `mode` also a parameter for this op? In the signature of the shape function it is equal to `0` as the default value, but in general it could have any of the three modes as its value. The same is true for all other parameters

Reply:
I wasn't sure its it was a parameter or a fixed value. If its a parameter I can reuse the same shape function I think

Reply:
Yeah, that would be great! Just refactor into a helper python function, then you can just use it for both

Reply:
yup makes sense